src/app/static/src/app/components/projects/Projects.vue

```diff
@@ -12,6 +12,7 @@
             </div>
             <div class="my-3 col-6 clearfix">
                 <input type="text" class="form-control"
+                       @keyup.enter="!disableCreate ? createProject(newProjectName) : ''"
```

Comment:
I could take this logic out of the html, but it would require me to add another method with the disableCreate condition

Reply:
Making this a statement rather than expression would make more sense (I don't think we care about returning a value):
```suggestion
                       @keyup.enter="!disableCreate && createProject(newProjectName)"
```
More generally (and I'm not suggesting making this change here!) this should probably be an HTML form. If it was the we'd get automatic form submission when enter was pressed (which we could prevent by the usual means in the form was invalid) and it would also benefit accessibility (by being more compatible with screen readers).